tests/benchmark/benchmark.js

```diff
@@ -0,0 +1,72 @@
+/* use production mode for benchmark*/
+const fs = require('fs');
+const resolve = require('resolve');
+const resolveOpts = {
+    basedir: process.cwd()
+};
+const reactPath = fs.realpathSync(resolve.sync('react', resolveOpts));
+const reactDomPath = fs.realpathSync(resolve.sync('react-dom/server', resolveOpts));
+const reactDistPath = fs.realpathSync(resolve.sync('react/dist/react.min', resolveOpts));
+const reactDistDomPath = fs.realpathSync(resolve.sync('react-dom/dist/react-dom-server.min', resolveOpts));
+
+require(reactDistPath);
+require.cache[reactPath] = require.cache[reactDistPath];
+require(reactDistDomPath);
+require.cache[reactDomPath] = require.cache[reactDistDomPath];
+
+/* start with benchmark */
+const { Suite } = require('benchmark');
+const { renderToString } = require('react-dom/server');
+const { createElement, createClass, PropTypes } = require('react');
+const createI13nNode = require('../../dist/utils/createI13nNode');
+const I13nMixin = require('../../dist/mixins/I13nMixin');
+
+var PureReactComponent = createClass({
+    autobind: false,
+    render: function () {
+        return createElement('a', this.props, this.props.children);
+    }
+});
+
+var PureReactComponentWithI13n = createClass({
+    autobind: false,
+    mixins: [I13nMixin],
+    render: function () {
+        return createElement('a', this.props, this.props.children);
+    }
+});
+
+var createLinkChildren = function (Element, count) {
+    var links = [];
+    for (let i = 0; i < count; i ++) {
+        links.push(createElement(Element, {
+            key: i,
+            href: 'http://www.yahoo.com/' + i
+        }, 'text ' + i));
+    }
+    return createElement('div', {}, links);
+};
+
+var I13nA = createI13nNode('a');
+
+new Suite()
+.add('link-without-react-component', () => {
+    renderToString(createLinkChildren('a', 1));
+})
+.add('link-wrapped-with-react-component', () => {
+    renderToString(createLinkChildren(PureReactComponent, 1));
+})
+.add('link-wrapped-with-react-component-with-i13n-mixin', () => {
+    renderToString(createLinkChildren(PureReactComponentWithI13n, 1));
+})
+.add('link-wrapped-with-react-component-with-i13n-high-order-component', () => {
+    renderToString(createLinkChildren(I13nA, 1));
```

Comment:
I think u should replace `I13nA` with the `I13nAnchor` component cause it's the prominent use case, also throw in `I13nButton` too?